packages/mapping/ol/mapmodule/bundle.js

```diff
@@ -474,6 +474,12 @@ Oskari.clazz.define(
                 }, {
                     'type': 'text/javascript',
                     'src': '../../../../bundles/mapping/mapmodule/request/MapTourRequestHandler.js'
+                }, {
+                    'type': 'text/javascript',
+                    'src': '../../../../bundles/mapping/mapmodule/request/SetTimeRequest.js'
+                },{
+                    'type': 'text/javascript',
+                    'src': '../../../../bundles/mapping/mapmodule/request/SetTimeRequestHandler.js'
```

Comment:
event missing from here (if the notifyTimeChanged() is in this bundle the event should be included as well).